sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_file_system_client.hpp

```diff
@@ -108,86 +108,89 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
+        const DeleteDataLakeFileSystemOptions& options = DeleteDataLakeFileSystemOptions()) const;
 
     /**
      * @brief Sets the metadata of file system.
      * @param metadata User-defined metadata to be stored with the filesystem. Note that the string
      *                 may only contain ASCII characters in the ISO-8859-1 character set.
      * @param options Optional parameters to set the metadata to this file system.
-     * @return Azure::Core::Response<Models::SetFileSystemMetadataResult> containing the information
-     * returned when setting the metadata onto the file system.
+     * @return Azure::Core::Response<Models::SetDataLakeFileSystemMetadataResult> containing the
+     * information returned when setting the metadata onto the file system.
      * @remark This request is sent to blob endpoint.
      */
-    Azure::Core::Response<Models::SetFileSystemMetadataResult> SetMetadata(
+    Azure::Core::Response<Models::SetDataLakeFileSystemMetadataResult> SetMetadata(
         Storage::Metadata metadata,
-        const SetFileSystemMetadataOptions& options = SetFileSystemMetadataOptions()) const;
+        const SetDataLakeFileSystemMetadataOptions& options
+        = SetDataLakeFileSystemMetadataOptions()) const;
 
     /**
      * @brief Gets the properties of file system.
      * @param options Optional parameters to get the metadata of this file system.
-     * @return Azure::Core::Response<Models::GetFileSystemPropertiesResult> containing the
+     * @return Azure::Core::Response<Models::GetDataLakeFileSystemPropertiesResult> containing the
      * information when getting the file system's properties.
      * @remark This request is sent to blob endpoint.
      */
-    Azure::Core::Response<Models::GetFileSystemPropertiesResult> GetProperties(
-        const GetFileSystemPropertiesOptions& options = GetFileSystemPropertiesOptions()) const;
+    Azure::Core::Response<Models::GetDataLakeFileSystemPropertiesResult> GetProperties(
+        const GetDataLakeFileSystemPropertiesOptions& options
+        = GetDataLakeFileSystemPropertiesOptions()) const;
 
     /**
      * @brief List the paths in this file system.
      * @param recursive If "true", all paths are listed; otherwise, only paths at the root of the
      *                  filesystem are listed. If "directory" is specified, the list will only
      *                  include paths that share the same root.
      * @param options Optional parameters to list the paths in file system.
-     * @return Azure::Core::Response<Models::ListPathsResult> containing the results when listing
-     * the paths under a file system.
+     * @return Azure::Core::Response<Models::ListDataLakePathsResult> containing the results when
+     * listing the paths under a file system.
      * @remark This request is sent to dfs endpoint.
      */
-    Azure::Core::Response<Models::ListPathsResult> ListPaths(
+    Azure::Core::Response<Models::ListDataLakePathsResult> ListPaths(
```

Comment:
`SinglePage`? why it's not done in https://github.com/Azure/azure-sdk-for-cpp/pull/1259?

Reply:
Oops, must have missed that. Do you want me to fix it in this PR?

Reply:
single-purpose please.

Make sure you didn't miss the others

Reply:
Will be changed in a separate PR.